static/docs/commands-reference/get-url.md

```diff
@@ -27,6 +27,8 @@ directory.
 Note that this command doesn't require an existing DVC project to run in. It's a
 single-purpose command that can be used out of the box after installing DVC.
 
+> See `dvc get` to download data from other DVC repositories (e.g. Github URLs).
```

Comment:
in this specific case I would mention everywhere `data or model file or directory`. We want for ppl to pattern match on the case when model is needed in production environment.

Reply:
Understood. Updated in 2b1008a